rca/programmes/models.py

```diff
@@ -98,7 +100,8 @@ class ProgrammePage(BasePage):
     related_content_title = models.CharField(
         blank=True,
         max_length=120,
-        help_text="Large title displayed above the related content items, EG 'More opportunities to study at the RCA'",
+        help_text="Large title displayed above the related content items, \
+            e.g. 'More opportunities to study at the RCA'",
```

Comment:
```
help_text="Large title displayed above the related content items, " \
                 "e.g. 'More opportunities to study at the RCA'",
```

isn't that what you want? Your way will add all the spaces between `e.g.` and the rest of the text.

Reply:
yea you are right, looking at the label it renders ok but the html is full of extra spaces, I'll correct this